src/web_ui/__init__.py

```diff
@@ -1,27 +1,38 @@
 # Set up logging
 import logging
 
-from multiprocessing.sharedctypes import Value
+from constants import NAME_KEY_MAP
 from flask import Flask, render_template, request
-
 from node_mgmt import NetworkEnv, EnvScanner, get_ssh_fingerprint, Node
 from node_mgmt.commands import (
     imt_sensor_address,
     holykell_sensor_address_7,
     holykell_sensor_address_8,
     trigger_config_generation
 )
+
+
 import os
+import json
+import datetime
+
 from urllib.request import urlopen
 from kvstore import keys, KVStore
+from reader.mqtt_reader import Reader
 
-logging.basicConfig(format='%(name)s [%(levelname)s] %(message)s', level='INFO')
+logging.basicConfig(
+    format='%(name)s [%(levelname)s] %(message)s', level='INFO')
 logger = logging.getLogger(__name__)
 
 app = Flask(__name__)
 
 kvs = KVStore()
 
+app.run(host="0.0.0.0", debug=True)
+
+reader = Reader()
+reader.__enter__() # shouldn't call __enter__ manually, __exit__ wasnn't called
```

Comment:
Your comment is correct. The way to actually instantiate and use these is
```
with Reader() as reader:
    result_bytes = reader.read('u/data')
... do something with result_bytes
```
This will take care of session management properly.

There's also no reason to be instantiating `reader` and creating a global session here, since this is only actually used inside the `realtime_readings()` function.

Reply:
I tried that way of instantiate the reader before but it wasn't working well when I constantly refresh the page for updates of readings. We could talk about this later during the call